arch/risc-v/src/mpfs/mpfs_coremmc.c

```diff
@@ -45,7 +45,7 @@
 
 #include <arch/board/board.h>
 
-#include "mpfs_emmcsd.h"
+#include "mpfs_coremmc.h"
 #include "riscv_internal.h"
 #include "hardware/mpfs_coremmc.h"
 
```

Comment:
You need mpfs_sdio_dev.h here ?

struct mpfs_dev_s is now defined in two places:
- mpfs_sdio_dev.h
- mpfs_coremmc.c

How should this work ?

Reply:
Right.. changes to mpfs_coremmc.c were accidentally left out from commit. Need to fix